playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap

```diff
@@ -28,7 +28,7 @@ exports[`html structure is correct 1`] = `
               style="vertical-align: middle; color: rgb(193, 205, 214);"
             >
               <i
-                class="pb_icon_kit far fa-fw fa-lg fa-chevron-down"
+                class="pb_icon_kit far far fa-lg fa-fw fa-fw fa-lg fa-chevron-down"
```

Comment:
@markdoeswork Why are the classes duplicated on these tests?

Reply:
you know i never really looked into it, Ill go through that again now